.github/workflows/sunnypilot-master-dev-c3-prep.yaml

```diff
@@ -151,9 +151,10 @@ jobs:
                   }
                 }
               }
-            }' -F label="is:pr is:open label:${PR_LABEL} draft:false sort:created-asc")
+            }' -F label="is:pr is:open label:${PR_LABEL} draft:false sort:created-asc" \
+            > /tmp/pr_list_output.json
 
-          echo "PR_LIST=${PR_LIST}" >> $GITHUB_OUTPUT
+          echo "PR list written to /tmp/pr_list_output.json"
```

Comment:
Instead of writing this to tmp, it's better if you base64 encode it to prevent issues with characters and then decode it on the next step